packages/svelte-docgen/src/parser/mod.js

```diff
@@ -1,5 +1,36 @@
 /**
- * @import * as Doc from "../doc/type.ts";
+ * TODO: dts-buddy does not yet support `import * as Doc from "../doc/type.ts";`
+ * @import {
+ *	 Conditional,
+ *	 Fn,
+ *	 Index,
+ *	 IndexedAccess,
+ *	 Interface,
```

Comment:
dts-buddy doesn't seem to handle `@import * as Doc from "../doc/type.ts"`

Reply:
Hm, that's probably the reason it was declared as namespace previously. I followed the pattern from svelte repository.

But one of ESLint's rules/recommendations was to not use it.